### DISCUSSION TITLE
GTK3 vs GTK4 version

### DISCUSSION THREAD

**Discussion by mizaki** [GeoRW/ACBF-Editor#7] [General]
It maybe that we can make a GTK3 branch for your update and GTK4 for mine? Get the GTK3 releasable and move all new work to the GTK4 as master.

**Comment by GeoRW** [comment]
Yes, that's what I was thinking too. Release GTK3 version and then switch to GTK4. I will need to upgrade my system though as my GTK 4 is only version 4.6.9 and your code needs at least 4.10. Does my code works on your system now?

**Comment by GeoRW** [comment]
So, I created release/v3.0 branch. So, if you want to may import your stuff into main

**Comment by mizaki** [comment]
I've replaced main with my GTK4 main.
